nc-news/src/Components/Articles.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { ArticleCard } from "./ArticleCard";
+
+export const Articles = () => {
+    const [articles, setArticles] = useState([]);
+
+    useEffect(() => {
+        axios.get("https://robbies-articles-website.onrender.com/api/articles")
```

Comment:
Please consider separating out your api requests into their own file -  and also take a look at the axios.create method to make your code even DRYer!

Reply:
Please include a loading pattern to improve the user experience while they’re waiting for the page to render.